api/endpoint-tests/endpoint.sh

```diff
@@ -4,22 +4,24 @@ rm ./endpoint-tests/endpoint-data.json
 
 export NODE_ENV=test
 export API_URL=http://localhost:8081
+export COMPOSE_PROJECT_NAME=api
 
 unset DEV_DB_NAME
 
 echo "[]" > endpoint-data.json
 
-docker-compose -f docker-compose.endpoint-tests.yml up -d
-sleep 3
-docker cp ./ api-for-testing:/app/ 
-docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing yarn run migrate
-docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing yarn run seed
-docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing yarn run test-endpoints $@
+docker-compose -f ../docker-compose.endpoint-tests.yml up -d
+
```

Comment:
Docker CP is not needed here?

Reply:
I don't really understand the question, but we do need to use docker compose in this script

Reply:
Resolved in Slack convo with Tiff